pkgs/desktops/gnome/default.nix

```diff
@@ -339,7 +339,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   inherit (pkgs) gnome-user-docs; # added 2019-11-20
 
-  inherit (pkgs) gegl_0_4; # added 2019-10-31
+  inherit (pkgs) gegl; # added 2019-10-31
```

Comment:
This was just an alias for compatibility, changing it will remove the alias so we can just remove this altogether.

Reply:
Addressed in 4cea628efbd190a0678a61bd93f87829c0fbcae0